charmed_openstack_info/data/lp-builder-config/openstack.yaml

```diff
@@ -1817,6 +1988,15 @@ projects:
         bases:
           - "22.04"
           - "23.04"
+      stable/2023.2:
+        enabled: True
+        build-channels:
+          charmcraft: "2.2/stable"
+        channels:
+          - 2023.2/candidate
+        bases:
+          - "22.04"
+          - "23.10"
 
   # - name: OpenStack Neutron Dynamic Routing Charm
```

Comment:
A rebase is needed after #40 merged, it provided a commented out section for the stable channel, which I guess can be uncommented as part of this PR